### PR TITLE
Revise Concat op - specification

### PR DESCRIPTION
**Pull request by blesniewski**
### Details:
 - Clarified on the `axis` attribute range
 - Added second example to show usage of a negative axis index

### Tickets:
 - *37427*

### CODE REVIEW

**Review by jdanieck — APPROVED** [review]
Please add another example with negative axis.

**Comment by jdanieck** [comment]
Note: we are missing 'detailed description' section, but I don't have an idea what we should put there since this is so simple operator :)

File: docs/ops/movement/Concat_1.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 * *axis*
 
   * **Description**: *axis* specifies dimension to concatenate along
-  * **Range of values**: integer number. Negative value means counting dimension from the end
+  * **Range of values**: integer number. Negative value means counting dimension from the end. Range is '[-R, R-1]' where 'R' is the rank of all inputs.
```
Comment (andrew-zaytsev) [inline docs/ops/movement/Concat_1.md:14]: Minor edits are required:

Range is '[-R, R-1]' where 'R' is the rank of all inputs. --> The range is '[-R, R-1]', where 'R' is the rank of all inputs.

**Review by andrew-zaytsev — CHANGES_REQUESTED** [review]

**Comment by blesniewski** [comment]
@andrew-zaytsev Can you take a look at this PR?

File: docs/ops/movement/Concat_1.md [hunk 2]
```diff
@@ -11,7 +11,7 @@
 * *axis*
 
   * **Description**: *axis* specifies dimension to concatenate along
-  * **Range of values**: integer number. Negative value means counting dimension from the end
+  * **Range of values**: integer number. Negative value means counting dimension from the end. The range is '[-R, R-1]', where 'R' is the rank of all inputs.
```
Comment (ilyachur) [inline docs/ops/movement/Concat_1.md:14]: @lazarevevgeny Please have a look

Comment (lazarevevgeny) [reply docs/ops/movement/Concat_1.md:14]: Use back-tick character to wrap all formulas. "`"

Comment (blesniewski) [reply docs/ops/movement/Concat_1.md]: Done.

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**